pkgs/applications/editors/vim/macvim.nix

```diff
@@ -191,12 +212,21 @@ stdenv.mkDerivation (finalAttrs: {
     libperl=$(dirname $(find ${perl} -name "libperl.dylib"))
     install_name_tool -add_rpath ${luajit}/lib $exe
     install_name_tool -add_rpath ${tcl}/lib $exe
+  ''
+  + optionalString enablePython ''
     install_name_tool -add_rpath ${python3}/lib $exe
+  ''
+  + ''
     install_name_tool -add_rpath $libperl $exe
     install_name_tool -add_rpath ${ruby}/lib $exe
 
     # Remove manpages from tools we aren't providing
-    find $out/Applications/MacVim.app/Contents/man -name evim.1 -delete
+    find $out/Applications/MacVim.app/Contents/man \( -name evim.1 -or -name eview.1 \) -delete
+    rm $out/Applications/MacVim.app/Contents/man/man1/mvim.1
+  ''
+  + optionalString stdenv.isAarch64 ''
+    # Resign the binary and set the linker-signed flag.
+    rcodesign sign --code-signature-flags linker-signed $exe
```

Comment:
Is there a reason not to use `autoSignDarwinBinariesHook` here?

Reply:
The answer is that I didn't know about that hook. That said, we only need to resign one binary, and it looks like that hook is going to check every single file to see if it needs a signature, so that's going to end up doing a lot more work for the same effect.